packages/subgraph/tests/utils/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const ADDRESS_ZERO = '0x0000000000000000000000000000000000000000';
+export const ADDRESS_ONE = '0x0000000000000000000000000000000000000001';
+export const ADDRESS_TWO = '0x0000000000000000000000000000000000000002';
+export const ADDRESS_THREE = '0x0000000000000000000000000000000000000003';
+export const ADDRESS_FOUR = '0x0000000000000000000000000000000000000004';
+export const ADDRESS_FIVE = '0x0000000000000000000000000000000000000005';
+export const ADDRESS_SIX = '0x0000000000000000000000000000000000000006';
+export const DAO_ADDRESS = '0x00000000000000000000000000000000000000da';
+export const CONTRACT_ADDRESS = '0x00000000000000000000000000000000000000Ad';
+export const DAO_TOKEN_ADDRESS = '0x6B175474E89094C44Da98b954EedeAC495271d0F';
+
+export const PLUGIN_SETUP_ID =
+  '0xfb3fd2c4cd4e19944dd3f8437e67476240cd9e3efb2294ebd10c59c8f1d6817c';
+export const APPLIED_PLUGIN_SETUP_ID =
+  '0x00000000cd4e19944dd3f8437e67476240cd9e3efb2294ebd10c59c8f1d6817c';
```

Comment:
What are those addresses for? Are they random? Maybe an explanation would be good